localstack/services/sqs/sqs_starter.py

```diff
@@ -65,6 +65,9 @@ def add_message(self, *args, **kwargs):
     def _set_attributes(self, attributes, now=None):
         _set_attributes_orig(self, attributes, now)
 
+        if attributes == {}:
```

Comment:
Thanks for digging into this, @mgagliardo ! 👍 

Not entirely sure if this is the right approach here. Looking at the definition of `sqsQueueEmptyPolicyAttributes` [here](https://github.com/hashicorp/terraform-provider-aws/blob/ccc13f55709c860a2130ecee6e21acde2318e936/aws/resource_aws_sqs_queue_policy.go#L17-L21) and how it is being used in that file, I think we should rather check if `attributes` contains a `Policy` attribute with an empty string:

```
if attributes.get('Policy') == '':
    self.policy = None
```

Can you please double-check if this change fixes the issue with Terraform? Thanks!

Reply:
I have already checked that and sadly when you send an empty attribute (i.e. `--attributes Policy=""` or even trying a terraform destroy to trigger this) :

Just a simple log, when creating the policy (setting attributes):
```
Attributes passed: {'ContentBasedDeduplication': 'false', 'DelaySeconds': '0', 'FifoQueue': 'false', 'KmsDataKeyReusePeriodSeconds': 300, 'KmsMasterKeyId': None, 'MaximumMessageSize': '262144', 'MessageRetentionPeriod': '345600', 'Policy': None, 'ReceiveMessageWaitTimeSeconds': '0', 'RedrivePolicy': None, 'VisibilityTimeout': '30'}

Attributes passed: {'Policy': '{\n  "Version": "2012-10-17",\n  "Id": "__default_policy_ID",\n  "Statement": [\n    {\n      "Sid": "__default_statement_ID",\n      "Effect": "Allow",\n      "Action": "sqs:SendMessage",\n      "Resource": "arn:aws:sqs:us-east-1:000000000000:distributor-queue",\n      "Principal": {\n        "AWS": "*"\n      },\n      "Condition": {\n        "ArnEquals": {\n          "aws:SourceArn": [\n            "arn:aws:s3:::local-test-events",\n            "arn:aws:s3:potato"\n          ]\n        }\n      }\n    }\n  ]\n}'}
```

And then when destroying:
```
Attributes passed: {}
```